src/interfaces/IConditionalOrder.sol

```diff
@@ -30,6 +31,7 @@ interface IConditionalOrder {
      * @param sender the `msg.sender` of the transaction
      * @param _hash the hash of the order
      * @param domainSeparator the domain separator used to sign the order
+     * @param ctx the context of the order (bytes32(0) if a merkle tree is used, otherwise H(params))
```

Comment:
Maybe specify the intent here that this value can be used to look up order specific storage in composable cow?

Reply:
Done